openbb_platform_pro_backend/main.py

```diff
@@ -11,6 +13,22 @@
     data_schema_to_columns_defs,
 )
 
+CURRENT_USER_SETTINGS = os.environ.get("HOME") + "/.openbb_platform/user_settings.json"
```

Comment:
to avoid potential errors on windows let's specify the path either with `Path` or as `os.path.join`